### ISSUE TITLE
Failed to install on go1.7.1 linux/386

### ISSUE THREAD

**Issue by muaazsaleem** [johnnadratowski/golang-neo4j-bolt-driver#16]
Hey Guys,

I am _go installing_ with go1.7.1 linux/386 and I get the following error messages:

```
# github.com/johnnadratowski/golang-neo4j-bolt-driver/encoding
encoding/encoder.go:194: constant 9223372036854775807 overflows int
encoding/encoder.go:328: constant 4294967295 overflows int
encoding/encoder.go:363: constant 4294967295 overflows int
encoding/encoder.go:405: constant 4294967295 overflows int

```

**Comment by johnnadratowski** [comment]
@muaazsaleem - Sorry for the late reply, I've had a lot of things in my personal life going on.

I submitted a PR for this issue: https://github.com/johnnadratowski/golang-neo4j-bolt-driver/pull/21

This should resolve the issues you have installing on a 32 bit machine.

Thanks for the issue submission! After the CI tests pass I'm going to test it in a local VM of Ubuntu Xenial 32 bit.  I've already tested it by locally building, but I'll retest after I merge.
